File: inputprint.ts
// inputprint.ts
import * as readline from 'readline';

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question('Enter some text: ', (input) => {
    console.log(`You entered: ${input}`);
    rl.close();
});

File: inputprint.rs
// inputprint.rs
use std::io::{self, Write};

fn main() {
    print!("Enter some text: ");
    io::stdout().flush().unwrap();

    let mut input = String::new();
    io::stdin().read_line(&mut input).unwrap();

    println!("You entered: {}", input.trim());
}
